src/services/email.service.ts

```diff
@@ -0,0 +1,26 @@
+import logger from "../utils/logger";
+
+require("dotenv").config();
+
+const sgMail = require("@sendgrid/mail");
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+export function sendConfirmationEmail(email: string) {
+  const message = {
+    to: email,
+    from: process.env.SENDGRID_FROM_ADDRESS,
+    subject: "Kid Toys account created",
+    text: "Thank you for creating an account at Kid Toys.",
+  };
+
+  sgMail
+    .send(message)
+    .then((response: any) => {
+      logger.info(response[0].statusCode);
+      logger.info(response[0].headers);
```

Comment:
You can add this in one logger.info statement. Something like "Email sent successfully" you don't need to log status code and headers.

Reply:
Fixed.